.rubocop.yml

```diff
@@ -45,6 +45,9 @@ Naming/MemoizedInstanceVariableName:
 Style/RedundantInitialize:
   Enabled: false
 
+Style/HashSyntax:
+  Enabled: false
```

Comment:
Any reason why we don't want this?  You can choose our preferred style by setting the `EnforcedStyle` ... https://www.rubydoc.info/gems/rubocop/0.46.0/RuboCop/Cop/Style/HashSyntax

I would think `no_mixed_keys` gives a pretty good result as we shouldn't be mixing key types.

Reply:
Okay, I will change that.